### ISSUE TITLE
Show the indicator in the channel list for mentions in a focused room

### ISSUE THREAD

**Issue by tomrittervg** [element-hq/element-web#13275]
**Is your suggestion related to a problem? Please describe.**
I switch to Riot and I am unsure if I have a mention in the presently focused room, because the indicator in the channel list does not appear for the focused room.

**Describe the solution you'd like**
If the Riot app (I used the electron app) or the Riot tab is not the focused window; I'd like to see the red indicator in the channel list, and for it to stay there until I have scrolled up and seen the indicated message.  If there are more than one, it would decrement for each one I view but not go away until I see them all.

**Describe alternatives you've considered**
Presently I will focus on a disused room I don't expect a mention in.

**Comment by jryans** [comment]
I agree we consider an option for something like this, as I also have taken to changing to a disused room to avoid notification clearing / read marker advancing, which is quite sad, as it feels like I am working around Riot's behaviour.

We'll be looking at notifications holistically soon, so we can consider this dilemma as part of it.

**Comment by turt2live** [comment]
ftr we made this worse with the new room list in that it actively ignores these sorts of updates on the selected room.

**Comment by florianduros** [comment]
Now the room list displays the mention indicator even if the room is currently opened
